src/main/java/org/opentripplanner/middleware/utils/JsonUtils.java

```diff
@@ -72,6 +74,36 @@ public static void logMessageAndHalt(Request request, int statusCode, String mes
         return null;
     }
 
+    /**
+     * Check if an {@link HttpResponse} is OK.
+     */
+    private static boolean isResponseOk(HttpResponse<String> response) {
```

Comment:
Suggestion: rename to `isNotError`. Remove redundant comment, or state in comment what constitutes a good vs. bad response.